planner/start_server.py

```diff
@@ -15,44 +17,64 @@
 _ONE_DAY_IN_SECONDS = 60 * 60 * 24
 
 
+def routeresults_to_json(list: List[RouteResult]) -> str:
```

Comment:
Non canonical style, subclass `JSONEncoder` and pass it into `json.dumps` instead.

Reply:
This is pretty tricky...https://stackoverflow.com/questions/5906831/serializing-a-python-namedtuple-to-json